modules/javafx.controls/src/test/java/test/javafx/scene/chart/CategoryAxisTest.java

```diff
@@ -118,10 +117,6 @@ public CategoryAxisTest() {
     }
 
 
-    @Test public void checkCategorySpacingReadOnlyCannotBind() {
-        assertTrue(axis.categorySpacingProperty() instanceof ReadOnlyDoubleProperty);
```

Comment:
In theory, this should be replaced with a `null` check, but if it's `null` the test will fail anyway, so it should be fine.

Reply:
I wonder if this change should be reverted. Since this is in a test, there is value in ensuring that what is true today remains true. This will ensure that the test will fail if the return type of this property ever changed to make the `instanceof` check no longer pass. Now admittedly, this is unlikely, since it would be a breaking change that should be caught during a code review, but it does have some (small) value.

Reply:
I could replace it with a class check instead.  I was also debating whether this really has any value when I removed it.